R/accessory.R

```diff
@@ -889,6 +889,16 @@ checkListIsSubset <- function(test_list,
 
 #' Read and validate a contrasts file against sample metadata
 #'
+#' Checks: 
+#' 1. No duplicate contrast IDs. Ensure that the required columns (variable, reference, target) are present.  
+#' 2. Values in the contrast variable column exist as column names in the sample metadata.
+#' 3. If blocking factors are supplied, checks that they are present in the sample metadata.
+#' 4. Design matrix is full rank.
+#' 5. Warn about continuous covariates (e.g. numeric patient IDs treated as continuous).
+#' 6. Values of specified columns don't contain special characters that the pipeline can't handle.
```

Comment:
```suggestion
#' 6. Values of specified columns don't contain special characters.
```
This code isn't necessarily being used in a pipeline